webui/src/App.js

```diff
@@ -21,86 +21,168 @@ import {
   Link
 } from "react-router-dom";
 
-// Amount of panels.
-const num_panels = 4;
+// Maximum number of historical sensor values to retain
+const max_size = 1000;
```

Comment:
Can we rename to `MAX_SIZE` (even though I know the originally it was `max_size`).

To answer your question in the other comment. The `k` prefix was what's used to denote constants in C++ as part of the Google Style Guide. I copied that paradigm for the JS here, but reading up on Google's style guide for JS, the actually use CONSTANT_CASE, so I'm going to suggest that moving forward:

https://google.github.io/styleguide/jsguide.html#naming-constant-names

I'll run a linter on this in a follow up and fix all the the things I've messed up, but I think tackling the smaller one within this PR sgtm.

Reply:
> Can we rename to MAX_SIZE (even though I know the originally it was max_size).

Will do.

> The k prefix was what's used to denote constants in C++ as part of the Google Style Guide.

Ahh makes sense.